paddle/fluid/ir/dialect/paddle_dialect/ir/pd_manual_api.cc

```diff
@@ -18,5 +18,16 @@
 #include "paddle/ir/core/builtin_op.h"
 
 namespace paddle {
-namespace dialect {}  // namespace dialect
+namespace dialect {
+ir::OpResult split_grad(std::vector<ir::OpResult> out_grads,
```

Comment:
```suggestion
ir::OpResult split_grad(const std::vector<ir::OpResult>& out_grads,
```
@0x45f 我看 pd_api.h 里的 concat、add_n 的vector 入参也是值copy，这个是不是可以优化为 const &？

Reply:
这个可以在自动生成代码中完善